packages/interbit-covenant-tools/src/rootCovenant/covenants/manifestCovenant/applyJoinChanges.js

```diff
@@ -0,0 +1,170 @@
+const _ = require('lodash')
+const {
+  startConsumeState,
+  startProvideState,
+  authorizeReceiveActions,
+  authorizeSendActions
+} = require('interbit-covenant-utils')
+const {
```

Comment:
These action creators are exposed at '../../../coreCovenant/actionCreators, so you can merge these two requires.